config/schemas/config.yaml

```diff
@@ -8454,6 +8454,14 @@ properties:
             patternProperties:
               "^[a-z-]+$":
                 $ref: '#/$defs/containerImage'
+          configurations:
+            title: Crossplane Configuration images configuration
+            description: Crossplane Configuration images configuration
```

Comment:
Aren't these packages rather? 🤔 

(After some digging, yes they are not images even on a technical level as it doesn't involve the OCI image spec, it is part of the distribution spec for generic artefacts. 😅)